arches/app/models/querysets/graph.py

```diff
@@ -9,7 +12,20 @@ def create(self, *args, **kwargs):
             "Use create_graph() to create new Graph instances with proper business logic."
         )
 
-    def create_graph(self, name="", user=None, is_resource=False):
+    def get_slug(self, name, is_resource):
+        if name:
+            slug = slugify(name, separator="_")
+        else:
+            if is_resource:
+                slug = "new_resource_model"
+            else:
+                slug = "new_branch"
+        existing_slugs = GraphModel.objects.all().values_list("slug", flat=True)
+        slug = make_name_unique(slug, existing_slugs, "_")
+
+        return slug
+
+    def create_graph(self, name="", slug=None, user=None, is_resource=False):
```

Comment:
Since we have the opportunity before v8 goes out, let's force everything except name to be keyword-only. If I looked at a call like `create_graph("foo", None, None, True)`, I would have no idea what was going on.
```suggestion
    def create_graph(self, name="", *, slug=None, user=None, is_resource=False):
```